src/Client.php

```diff
@@ -431,6 +432,6 @@ public function getMouse()
             throw new \RuntimeException(sprintf('"%s" does not implement "%s".', \get_class($this->webDriver), WebDriverHasInputDevices::class));
         }
 
-        return $this->webDriver->getMouse();
+        return new WebDriverMouse($this->webDriver->getMouse(), $this);
```

Comment:
Can you also add the return type to `getMouse()` please?

Reply:
Done.